frontend/src/services/movies.ts

```diff
@@ -40,10 +41,68 @@ const filterMovies = async (pageNumber: number, genreId?: number | ""): Promise<
     }
 };
 
+const getMovieById = async (id: number): Promise<IMovie> => {
+    try {
+        const response = await axios.get(`${API_URL}/${id}`);
+        const movie = response.data;
+        return movie as IMovie;
+    } catch (error) {
+        console.error('Error fetching movies:', error);
+        throw error;
+    }
+};
+
+const addReview = (movieId:number, rating:number, comment: string, createdOn: string,accountId:number,movieTitle:string) => {
+    return axios
+      .post(`${API_URL}/${movieId}/reviews`, {
```

Comment:
I believe that this endpoint will needs an authentication according to:https://github.com/SEP6-VIA/SEP6/pull/60/files